share/spice/transit/njt/transit_njt.js

```diff
@@ -13,7 +13,6 @@
             signal: 'high',
             data: api_result.routes,
             meta: {
-                heading: api_result.origin + " to " + api_result.destination,
```

Comment:
@gokul1794 this line shouldn't be totally removed, you just need to replace <pre>heading</pre> with <pre>primaryText</pre>, like it's done [here](https://github.com/duckduckgo/zeroclickinfo-spice/pull/1826/files#diff-f43444f36ddec5dfc14010d40ef963e1R263). Thanks!